packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -177,6 +188,25 @@
     <T.Panel>Content 2</T.Panel>
     <T.Panel>Content 3!</T.Panel>
   </Hds::Tabs>
+
+  <h4 class="dummy-h4">Pass in a function that gets called when a tab is clicked</h4>
+  {{! template-lint-disable no-unbalanced-curlies }}
```

Comment:
The linter was complaining because of the slash I used to escape this value:
`@onTabClick=\{{this.logClickedTab}}` is there a better work around? (I tried different combinations of slashes but they were rendering to the browser.)

Reply:
It's a tricky thing: you have to do
```
{{! prettier-ignore-start }}
{{! template-lint-disable no-unbalanced-curlies }}
```
and not 
```
{{! template-lint-disable no-unbalanced-curlies }}
{{! prettier-ignore-start }}
```

Let me know if it still doesn't work.

Reply:
Also, I _think_ you can add also the "Renders to:" like for the other examples.

Reply:
Thanks! I wasn't sure if adding the "renders to" made sense for that section since there's no visual difference but I can add it.

Reply:
When I move that line the linter starts complaining again and it tells me the following:
```
211:13  error  Argument @onTabClick should be either quoted or wrapped in mustaches  no-quoteless-attributes
211:25  error  Replace `\{{this.logClickedTab}}` with `"\{{this.logClickedTab}}"`  prettier
```

When I add the quotes like it says then I get a different linter error:
```
198:53  error  " is not a valid character within attribute names: (error occurred in 'an unknown module' @ line 198 : column 53)  undefined
```

I've been trying different combinations of quotes and slashes to try to satisfy the linter but not having any luck so far.

Reply:
Never mind, it finally  worked. I think the build was failing from some previous attempt and I didn't notice it at first. Thanks for your help.